### ISSUE TITLE
[Suggestion] Replace QTransmission with qBittorrent

### ISSUE THREAD

**Issue by notuxius** [netrunner-rolling/QA-Suggestions#16]
I think qBittorrent is better - it has more convenient interface and easy way to shutdown/suspend.. system on downloads completion

**Comment by NuLogicSystems** [comment]
I agree with you on this one, however this one would be up to Starbuck. It probably would need to be changed on both Manjaro and Debian based versions.
